packages/browser/README.md

```diff
@@ -0,0 +1,178 @@
+```ts
+const attributes: Record<string, unknown> = {
+    "release": "PROD",
+}
+
+// BacktraceClientOptions
+const options: BacktraceConfiguration = {
+    // Name of the website/application
+    name: "MyWebPage",
+    // Version of the website
+    version: "1.2.3",  
+    // Submission url
+    url: "https://submit.backtrace.io/<universe>/<token>/json",
+    userAttributes: attributes,
+}
+
+// Initialize the client
+const client = BacktraceClient.builder(options).build();
+```
+
+#### During Runtime
+Global attributes can also be set during the runtime once specific data has be loaded (e.g. a user has logged in).
+```ts
+client.addAttribute({
+    "clientID": "de6faf4d-d5b5-486c-9789-318f58a14476"
+})
+```
+
+#### Situationally in the BacktraceData or BacktraceReport
+When a BacktraceReport object is provided the attributes list can be directly modified.
+
+```ts
+const report: BacktraceReport = new BacktraceReport("My error message");
+report.attributes["myKey"] = "myValue";
+```
+
+### File Attachments
+***
+Backtrace has several ways in which files can be attached: when initalizing the BacktraceClient, updating the BacktraceClient, or  dynamically for specific reports.  When specifing attachments to the BacktraceClient, all files will be uploaded with each report.
+
+```js
+// Import the attachment types from @backtrace/browser with your favoriate package manager.
+import { BacktraceUint8ArrayAttachment, BacktraceStringAttachment  } from "@backtrace/browser";
+
+// BacktraceStringAttachment should be used for text object like a log file for instance
+const attachment1 = new BacktraceStringAttachment("logfile.txt", "This is the start of my log")
+
+// BacktraceUint8ArrayAttachment should be used for binary files
+const attachment2 = new BacktraceUint8ArrayAttachment("output.dmp", new Uint8Array(2));
+
+// Setup array of files to attach
+const attachments = [attachment1];
+
+// BacktraceClientOptions
+const options = {
+    // Name of the website/application
+    name: "MyWebPage",
+    // Version of the website
+    version: "1.2.3",  
+    // Submission url
+    url: "https://submit.backtrace.io/<universe>/<token>/json",
+    // Attach the files to all reports
+    attachments: attachments
+}
+
+// Later decide to add an attachment to all reports
+client.attachments.push(attachment2)
+
+// After catching an exception and generating a report
+try {
+    throw new Error("Caught exception!")
+} catch (error) {
+    const report = new BacktraceReport(error);
+    const attachment3 = new BacktraceStringAttachment("CaughtErrorLog", "some error logging data here");
+    report.attachments.push(attachment3);
+    client.send(report);
+}
+```
+
+### Breadcrumbs
+Feature in developement
+
+### Application Stability Metrics
```

Comment:
I think this makes sense to list this under features, even if it does just link to the object below. The description can also link to Sauce Docs.